transmute_core/object_serializers/attrs_serializer.py

```diff
@@ -0,0 +1,57 @@
+from .interface import ObjectSerializer
+from cattr import structure, unstructure
+from ..exceptions import SerializationException
+from cattr.vendor.typing import (
```

Comment:
haha, it has been used for to_json_schema method before, but i have removed that part.